ClientSide(FrontEnd)/clay_/src/components/Images.css

```diff
@@ -0,0 +1,263 @@
+.img1{
```

Comment:
use proper naming convention for the giving classNames

Reply:
i changed naming convention